module/lezioni.py

```diff
@@ -1,60 +1,202 @@
 # -*- coding: utf-8 -*-
 
-import json
-import requests
-import sqlite3
-import calendar
-import datetime
-import locale
-import time
+# Telegram
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup, Update
+from telegram.ext import CallbackContext, CallbackQueryHandler
 
-locale.setlocale(locale.LC_ALL, 'it_IT.utf8')
+# System libraries
+import logging
 
-def get_numero_giorno(giorno):
-    switcher = {
-        'lunedi':'1',
-        'martedi':'2',
-        'mercoledi':'3',
-        'giovedi':'4',
-        'venerdi':'5'
-    }
+# Modules
+from module.shared import check_log, send_message
+
+import sqlite3
+import re
 
-    return switcher.get(giorno, "Giorno non valido")
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO)
+logger = logging.getLogger(__name__)
 
-def get_numero_anno(anno):
+def get_nome_giorno(day):
     switcher = {
-        'primo':1,
-        'secondo':2,
-        'terzo':3
+        '1' : 'Lunedì',
+        '2' : 'Martedì',
+        '3' : 'Mercoledì',
+        '4' : 'Giovedì',
+        '5' : 'Venerdì'
     }
 
-    return switcher.get(anno, "Anno non valido, utilizzare primo/secondo/terzo")
-
-def lezioni_output(item):
-    output = "*Nome:* " + item["nome"]
-    output += "\n*Aula:* " + str(item["aula"])
-    output += "\n*Anno:* " + str(item["anno"]) + "\n"
+    return switcher.get(day, "Giorno non valido")
 
+# eliminate
+def lezioni_button():
```

Comment:
Is this still used somewhere or can just be removed?

Reply:
Nope, I completely forgot to delete this 😅
I don't even remember why I have commented but not actualy deleted this...